crates/re_types/definitions/rerun/components/keypoint_id.fbs

```diff
@@ -19,13 +19,14 @@ namespace rerun.components;
 /// \rs
 /// \rs Used to look up an `crate::components::AnnotationInfo` for a Keypoint within the `crate::components::AnnotationContext`.
 struct KeypointId (
-  "attr.python.aliases": "float",
```

Comment:
:grimacing:

Reply:
yeah no idea where that came from. Suprised that this didn't cause some mypy issues?